tools/Microsoft.TemplateEngine.Authoring.TemplateVerifier/VerificationEngine.cs

```diff
@@ -341,8 +331,12 @@ private static CommandResultData RunDotnetNewCommand(TemplateVerifierOptions opt
                 cmdArgs.Add(options.TemplateName);
             }
 
-            var command = new DotnetCommand(loggerFactory?.CreateLogger(typeof(DotnetCommand)) ?? logger, "new", cmdArgs.ToArray())
-                    .WithWorkingDirectory(workingDir);
+            var command = new DotnetNewCommand(loggerFactory?.CreateLogger(typeof(DotnetCommand)) ?? logger, cmdArgs.ToArray())
+                .WithCustomOrVirtualHive(customHiveLocation)
```

Comment:
this looks weird a bit, should the handling rather be done in this method? 
On the other side it breaks decorators chain then.

Reply:
Yeah - I know. But it makes the code more conscise so in the end it's more understandable I believe